src/Microsoft.Sbom.Api/Config/ConfigSanitizer.cs

```diff
@@ -81,6 +82,20 @@ public IConfiguration SanitizeConfig(IConfiguration configuration)
         // Set default package supplier if not provided in configuration.
         configuration.PackageSupplier = GetPackageSupplierFromAssembly(configuration, logger);
 
+        // Prevent null value for LicenseInformationTimeoutInSeconds.
+        // Values from int.MinValue to int.MaxValue are allowed. Negative values are simply treated as InfiniteTimespan in LicenseInformationService, and
+        // from what I can tell there are no upper limits higher than int.MaxValue in Timespan.FromSeconds or httpClient.Timeout
+        if (configuration.LicenseInformationTimeoutInSeconds is null)
+        {
+            configuration.LicenseInformationTimeoutInSeconds = new(LicenseInformationFetcher.DefaultTimeoutInSeconds, SettingSource.Default);
+        }
+
+        // Check if arg -lto is specified but -li is not
+        if (configuration.FetchLicenseInformation?.Value != true && !configuration.LicenseInformationTimeoutInSeconds.IsDefaultSource)
+        {
+            logger.Warning("A license fetching timeout is specified (argument -lto), but this has no effect when FetchLicenseInfo is unspecified or false (argument -li)");
+        }
+
```

Comment:
This where we should enforce the limits that specify our allowed range of 1 to 86400, with unit tests to exercise the code paths